velox/functions/prestosql/benchmarks/IntArrayProxyBenchmark.cpp

```diff
@@ -206,6 +206,25 @@ class ArrayProxyBenchmark : public functions::test::FunctionBenchmarkBase {
     folly::doNotOptimizeAway(cnt);
   }
 
+  size_t runStdRef() {
+    for (auto k = 0; k < 100; k++) {
+      std::vector<std::vector<std::optional<int>>> arrayVector;
+      for (auto i = 0; i < 1000; i++) {
+        arrayVector.push_back({});
+        auto current = arrayVector[arrayVector.size() - 1];
+        for (int j = 0; j < i; j++) {
```

Comment:
again I feel like calling `current.resize(i)` or at least `current.reserve(i)` might be a more fair comparison, since the vector version does this.